internal/dependencymanager/dependencyinstaller.go

```diff
@@ -71,6 +71,10 @@ func (cl *categorizedLogs) LogAll(logger output.Logger) {
 		}
 		logger.Info("") // Add a line break after the section
 	}
+
+	if len(cl.fileSystemActions) == 0 && len(cl.stateUpdates) == 0 {
+		logger.Info("👍 Zero changes were made. Everything looks good.")
```

Comment:
In other places in the CLI, emojis are hidden on Windows because they weren't supported previously.  It looks like this may have changed in the last few years, but maybe we want to show this conditionally based on OS?  🤷 

https://github.com/onflow/flowkit/blob/6a5d2696cb01397c4d373628683f716ba65fccf5/output/emoji.go#L43